dev/source-release.sh

```diff
@@ -38,7 +38,7 @@ usage () {
 # Default repository remote name
 remote="apache"
```

Comment:
In the PR that updated `-g`, the default remote repository was changed from `origin` to `apache`.

I know many people use `apache` for their upstream project name.

But, I believe we used `origin` for the remote name as this is possibly expected to run on ASF infra / github runners, where the origin repository is `origin`.

If this runs on more than a PMCs laptop, then we should keep it as `origin` but add in the flag. This can be done in a follow up PR.